File: fib.js
function fib(n) {
  if (n < 2) {
    return n;
  }
  return fib(n - 2) + fib(n - 1);
}

console.log(fib(5));
console.log(fib(10));
console.log(fib(20));
console.log(fib(25));
console.log(fib(30));

File: fib.py
def fib(n: int):
    if n < 2:
        return n
    return fib(n - 2) + fib(n - 1)


print(fib(5))
print(fib(10))
print(fib(20))
print(fib(25))
print(fib(30))
